frontend/src/app/main-layout/sidebar/sidebar.component.ts

```diff
@@ -14,36 +14,51 @@ import { RouterLink, RouterOutlet } from '@angular/router';
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class SidebarComponent {
-  items: MenuItem[] = [
-    {
-      label: 'Component A',
-      route: '/component-a',
-    },
-    {
-      label: 'Component B',
-      route: '/component-b',
-    },
-    {
-      label: 'Products',
-      items: [
-        {
-          label: 'New',
-        },
-        {
-          label: 'Search',
-        },
-      ],
-    },
-    {
-      label: 'Profile',
-      items: [
-        {
-          label: 'Settings',
-        },
-        {
-          label: 'Logout',
-        },
-      ],
-    },
-  ];
+  auth = inject(AuthService);
+
+  items = computed<MenuItem[]>(() => {
+    return [
+      {
+        label: 'Component A',
```

Comment:
Should we keep the dummy components?

Reply:
I think think they can be removed now. I'll take them out in the next PR